tensorflow/python/ops/math_ops.py

```diff
@@ -100,6 +100,7 @@
 @@complex
 @@conj
 @@imag
+@@arg
```

Comment:
So should I move it or leave it here and then it will be moved when all others are moved.